UI/Controllers/ServersController.cs

```diff
@@ -59,19 +59,26 @@ public async Task<ActionResult> Edit(int id)
             }
 
             return View(serverForUpdating);
+
+            //else?
         }
         
         /// <summary>
         /// !REQUEST from updating form!
         /// </summary>
         /// <param name="server"></param>
-        [HttpPut]
-        public async Task<HttpStatusCode> Edit(Server server)
+        [HttpPost]
```

Comment:
For updating some model we should use PUT request, i think.

Reply:
attribute HttpPut doesn't work. and I received server's crash
meanwhile HttpPost works fine

Reply:
@alevel-petrenko , kk